src/index.html

```diff
@@ -11,6 +11,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="url" method="post"
```

Comment:
typically for method we use uppercase
```suggestion
    <form action="url" method="POST"
```